src/pages/components/CertificateViewDetails.js

```diff
@@ -276,9 +290,27 @@ const IconTime = styled.span`
   background: transparent url(${BgIconTimeSVG}) no-repeat;
 `
 
+const GreenText = styled.div`
+  color: #27847a;
+  font: normal 400 18.4px/0 Roboto;
+
+  @media (max-width: 1280px) {
+    font: normal 400 15.4px/0 Roboto;
+  }
+`
+
+const GreenBoldText = styled.div`
+  color: #27847a;
+  font: normal 700 18.4px/0 Roboto;
+  margin: 26px 0 8px 0;
+
+  @media (max-width: 1280px) {
+    font: normal 700 15.4px/0 Roboto;
+  }
```

Comment:
This isn't ideal. Why do we need to change the font on small screens? Also if we do I'd recommend swapping the logic so this is a `min-width` condition

Reply:
You are right ... that being said, I am going to remove this as it doesn't make that difference and if i remove this it will make the code easier to read